tests/websocket.spec.js

```diff
@@ -99,6 +99,117 @@ suite('WebsocketClient', () => {
     });
   });
 
+  test('subscribes to additional products', done => {
+    var client;
+    const server = testserver(++port, () => {
```

Comment:
Didn't we stop incrementing the port?

Reply:
True. This must have been left over from multiple rebases. I'll fix.